cmd/iam-object-store.go

```diff
@@ -433,7 +433,7 @@ var (
 func (iamOS *IAMObjectStore) listAllIAMConfigItems(ctx context.Context) (map[string][]string, error) {
 	res := make(map[string][]string)
 
-	for item := range listIAMConfigItems(ctx, iamOS.objAPI, iamConfigPrefix) {
+	for item := range listIAMConfigItems(ctx, iamOS.objAPI, iamConfigPrefix+SlashSeparator) {
```

Comment:
Need to remove leading slash in strings in lines 410-418 - IIUC the reason for the CI failure. I will take a fresh look and fix this tomorrow otherwise.